pkg/dependencies/factory.go

```diff
@@ -1049,6 +1049,7 @@ func (f *Factory) WithPackageControllerClient(spec *cluster.Spec, kubeConfig str
 			return err
 		}
 		f.dependencies.PackageControllerClient = curatedpackages.NewPackageControllerClient(
+			f.dependencies.Helm,
```

Comment:
Why do we have two helm dependencies being passed in here?

Reply:
The first is for installing helm charts, the second is for removing helm charts. They can require different parameters when they're instantiated, so you can't assume that you can use the same object to do both. Kinda annoying, but grants flexibility.

Reply:
Adding comments

Reply:
What's a case where different need to be specified? I am wondering if we should only stick to one right now if we don't have that use case for two different right now. Whoever needs two separate can then decide to separate this.

Reply:
Removing via followup. There was a case during development, but then that case was eliminated, so they can be collapsed into one now.